bin/blueoak-server.js

```diff
@@ -3,10 +3,21 @@
  * Copyright (c) 2015-2016 PointSource, LLC.
  * MIT Licensed
  */
+var parseArgs = require('minimist');
 var server = require('../');
 
+// parse arguments
+var argv = parseArgs(process.argv.slice(2));
+
+// convert mocks from CSV into an array
+var mocks = argv.mocks || argv.m;
+if (mocks) {
+    mocks = mocks.split(',');
+}
+
 server.init({
-    appDir: process.cwd()
+    appDir: process.cwd(),
+    mocks: mocks
```

Comment:
yeah ... and it looks like `test/bos-mocks` is the way to go since the template project has a `test` dir, not a `tests` dir

which makes me think:
1. should/can the mocks directory be configurable through?
2. can/should the list of mocks be provided through configuration?